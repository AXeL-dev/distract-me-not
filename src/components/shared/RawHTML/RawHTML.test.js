import { render } from "@testing-library/react";
import { RawHTML } from ".";

it('renders correctly', () => {
  const html = '<span>Hello world!</span>';
  const { asFragment } = render(<RawHTML>{html}</RawHTML>);
  expect(asFragment()).toMatchSnapshot();
});
