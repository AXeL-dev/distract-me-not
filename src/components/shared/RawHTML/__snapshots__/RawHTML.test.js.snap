// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<DocumentFragment>
  <span
    class=""
  >
    <span>
      Hello world!
    </span>
  </span>
</DocumentFragment>
`;
